boards/BMS/Inc/App/App_Accumulator.h

```diff
@@ -3,6 +3,12 @@
 #include "App_InRangeCheck.h"
 #include "App_SharedExitCode.h"
 
+typedef enum
+{
+    ACCUMULATOR_SEGMENT_0 = 0U,
+    NUM_OF_ACCUMULATOR_SEGMENTS,
```

Comment:
I think a comment within the enum suggesting to "Add more segments if needed" would be nice so that the user knows they can modify the enum

Reply:
This is kind of common practice so the comment is too verbose. The person working with the code should already know that they need to modify the enum such that `NUM_OF_ACCUMULATOR_SEGMENTS` matches the number of segments theyre working with